content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -29,9 +29,10 @@ Each entry defines a lookup name - a logical name for the resource - and a colle
 * AccountId - Used to override the global account id attribute, if this resource is deployed in a different AWS account.
 
 **Global attributes**
-The mapping file also supports three **required** global attributes:
 
-* AccountId - The default AWS account containing resources in this mapping file.
+The mapping file also supports one _optional_ and two **required** global attributes:
+
+* AccountId (optional) - The default AWS account containing resources in this mapping file.
```

Comment:
Not sure why region is required, but thats for another PR. Its just there as a helper metadata so you don't have to set it everywhere.

Reply:
If I remember it correctly, AWSCore is using that region information to construct the API Gateway endpoint for service API calls.

Reply:
Maybe change it to something like: "The default AWS account that is used to search for AWS resources referenced in your O3DE application code"?

This global account ID may not be the account that contains the resource in the resource mapping file since each resource may has it own account ID.